nio/responses.py

```diff
@@ -694,6 +694,39 @@ def from_dict(cls, parsed_dict):
         )
 
 
+@dataclass
+class RegisterInteractiveError(ErrorResponse):
+    pass
+
+
+@dataclass
+class RegisterInteractiveResponse(Response):
+    stages: List[str] = field()
+    params: Dict[str, Any] = field()
+    session: str = field()
+    completed: List[str] = field()
+    user_id: str = field()
+    device_id: str = field()
+    access_token: str = field()
+
+    @classmethod
+    @verify(Schemas.register_flows, RegisterInteractiveError)
+    def from_dict(cls, parsed_dict: Dict[Any, Any]):
+        # type: (...) -> Union[RegisterInteractiveResponse, RegisterInteractiveError]
+        for flow in parsed_dict["flows"]:
+            stages = [stage for stage in flow["stages"]]
```

Comment:
My knee-jerk is to use `itertools.chain` here, but I'm not certain if that would make it more readable or not. I'll think about it again later :^)

Reply:
I haven't actually used that before, personally, so I hadn't thought of that!